docs/docs/administration/customize.md

```diff
@@ -908,7 +908,9 @@ Alternatively, you can change the global default by adding an overwriting rule i
 beans.presentationService.defaultUploadedPresentation=${bigbluebutton.web.serverURL}/default.pdf
 ```
 
-You'll need to restart BigBlueButton after the change with `sudo bbb-conf --restart`.
+Please use the file name "default.pdf" if you would not like BigBlueButton to generate thumbnails from this file. The thumbnails will be used in some frontend apps, such as Greenlight v2.
```

Comment:
Hum, I did test it locally with a different name and it seems it worked as expected, in the `metadata.xml` the thumbnails' links were correctly set. Do you have any snippet of code you saw that it only selects files named `default.pdf`?

Reply:
@GuiLeme , yes, if you use other names than default.pdf, thumbnails will be created from this file even if you don't want them to be created. The script sees the filename as you find in the issue that will be closed by this PR.